api/HTMLFrameElement.json

```
@@ -379,7 +379,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": true
+            "deprecated": false
```

The entirety of the `<frame>` element and its API is deprecated, including this property. Don't bring it back 🙅‍♂️

```suggestion
            "deprecated": true
```